app/assets/stylesheets/sections/nav_sidebar.scss

```diff
@@ -109,8 +109,8 @@
     width: $sidebar_width;
 
     .nav-sidebar {
-      margin-top: 20px;
-      position: fixed;
+      margin-top: 10px;
+      position: relative;
```

Comment:
This is how I was able to display the expand button below the nav elements.

Since the wrapping `.sidebar-wrapper` wrapper around `.nav-sidebar`, I don't think this needs to be, too.